gym/lib/gym/options.rb

```diff
@@ -82,6 +82,11 @@ def self.plain_options
                                      env_name: "GYM_CODE_SIGNING_IDENTITY",
                                      description: "The name of the code signing identity to use. It has to match the name exactly. e.g. 'iPhone Distribution: SunApps GmbH'",
                                      optional: true),
+        FastlaneCore::ConfigItem.new(key: :skip_package_ipa,
+                                     env_name: "GYM_SKIP_PACKAGE_IPA",
+                                     description: "Should we skip packaging the ipa?",
+                                     is_string: false,
+                                     optional: true),
```

Comment:
Instead of `optional: true`, use `default_value: false`. Both work fine, but it's more clear I think 👍